docs/source/howto/workflows/pw/relax.md

```diff
@@ -2,4 +2,73 @@
 
 # `PwRelaxWorkChain`
 
-*To be added.*
+
+The `PwRelaxWorkChain` is designed to perform robust geometry relaxations of atomic structures with Quantum ESPRESSO’s `pw.x`. It combines automated restarts with built‑in error handling.
+
+---
+
+## Minimal example: build and submit
+
+```python
+from aiida import orm, load_profile
+from aiida_quantumespresso.workflows.pw.relax import PwRelaxWorkChain
+from aiida_quantumespresso.common.types import RelaxType
+from aiida.engine import submit
+from ase.build import bulk
+
+load_profile()
+
+# Load your pw.x code and structure
+code = orm.load_code('pw@localhost')
+structure = orm.StructureData(ase=bulk('Si', 'diamond', 5.4))
+
+# Get a builder with sensible default parameters from a predefined protocol.
+builder = PwRelaxWorkChain.get_builder_from_protocol(
+    code=code,
+    structure=structure,
+    protocol="fast",   # choose from: fast, moderate, precise
+    relax_type=RelaxType.POSITIONS_CELL,  # relax both positions and cell
+    options={"resources": {"num_machines": 2},
+             "max_wallclock_seconds": 7200}
+)
+
+# Optionally adjust calculation parameters
+builder.base_relax.pw.parameters['SYSTEM']['ecutwfc'] = 60.0
+builder.base_relax.pw.parameters['SYSTEM']['ecutrho'] = 300.0
+
+# Submit the work chain
+node = submit(builder)
+print(f"Launched {node.process_label}<{node.pk}>")
+```
+
+---
+
+## Inspecting results
+
+After the work chain finishes, the main outputs are:
+
+* `output_structure`: the relaxed structure.
+* `output_parameters`: a dictionary with key results from Quantum ESPRESSO (total energy, Fermi energy, etc.)
+
+```python
+from aiida.orm import load_node
+
+wc = load_node(node.pk)
+print("Work chain finished ok:", wc.is_finished_ok)
+
+# Get the relaxed structure
+if "output_structure" in wc.outputs:
```

Comment:
Why have the `if` statement here but not for the `output_parameters`? I understand you want to avoid having an error when the work chain is not `finished_ok`, but then this cell will still raise when you try to get the `outputs_parameters`.

This is of course why I initially used `run` in the `PwBaseWorkChain`, and suggested `wait = True` in [my comment here](https://github.com/aiidateam/aiida-quantumespresso/pull/1074#issuecomment-3277523544). But I think for the How-to we can expect the reader to be competent enough to understand that in case the workflow is not completed successfully, the outputs might be non-existent. So I'd vote to remove the `if`.